assignments/prototypes.js

```diff
@@ -15,6 +15,17 @@
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
 
+
```

Comment:
Great job on this prototype. All of the console logs pass and you seem to understand the material. Keep up the good work Robert.